_editor.cfg

```diff
@@ -34,11 +34,6 @@
         append=yes
         title= _ "FtF - Epilogue"
     [/music]
-    [music]
-        name=Flight_of_the_Drakes.ogg
-        append=yes
-        title= _ "FtF - Flight of the Drakes"
-    [/music]
```

Comment:
why remove this?

Reply:
ah wait never mind, I see now, you changed it to pull from UMC Book 10 instead...